### PR TITLE
Add blogpost on Signalen's incubation

### LINKED ISSUE #149: Make a blog post about Signalen going to incubation

### PR DESCRIPTION
**Pull request by Ainali**
Fixes #149

-----
[View rendered _posts/2021-02-04-signalen-is-in-incubation.md](https://github.com/publiccodenet/blog/blob/ja-signalen-incubation/_posts/2021-02-04-signalen-is-in-incubation.md)

### CODE REVIEW

**Review by felixfaassen — CHANGES_REQUESTED** [review]
I think it is important to clarify what incubation means and how this relates to the commitment VNG has made of doing taking the lead in doing the stewarding work. Essentially they decided the VNG realisatie will commit to lead the community building and stewarding efforts in scaling up signalen. In order not to confuse people in the community and the people we work with at VNG I think we should add the position of VNG and how we help them with that effort.

**Comment by ElenaFdR** [comment]
@felixfaassen Is this explained publicly anywhere that we can link to, or would our blogpost also be the first English explanation of the VNG/VNG-R in the Dutch codebase ecosystem?

**Comment by felixfaassen** [comment]
B

> @felixfaassen Is this explained publicly anywhere that we can link to, or would our blogpost also be the first English explanation of the VNG/VNG-R's role in the Dutch codebase ecosystem?

@ElenaFdR @Ainali not yet as they are using the Signalen project as a try-out but Jacco is communicating it to the community.

**Comment by Ainali** [comment]
I and Felix had a chat, and the blog post as it is works, since it is very clear on what incubation in this specific case means. Bearing that in mind, it's important that the language used when promoting the blog post also is clear about it is incubation for the Standard for Public Code we are referring to.

**Review by felixfaassen — APPROVED** [review]

File: _posts/2021-02-04-signalen-is-in-incubation.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+# Signalen is in incubation! Hurrah!
+
+![Signalen logo]({{site.url}}/assets/signalen_logo.png)
+
+We are delighted to announce that the [Signalen codebase](https://signalen.org/en/) is now in incubation with the [Foundation for Public Code](https://publiccode.net).
+
+During [incubation](https://about.publiccode.net/activities/codebase-stewardship/lifecycle.html), a codebase community works to make the codebase fully [Standard for Public Code](https://standard.publiccode.net/) compliant (with our support).
+
+This is an important milestone in the Signalen community's ambition, reflecting their strong commitment to openness and collaboration.
```
Comment (ElenaFdR) [inline _posts/2021-02-04-signalen-is-in-incubation.md:19]:
```suggestion
This is an important milestone in the Signalen community's ambitions, reflecting their strong commitment to openness and collaboration.
```

File: _posts/2021-02-04-signalen-is-in-incubation.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+After nearly a year in assessment, we're honored they've chosen to work more closely with us in the future, and proud the Standard has proven useful for the Signalen community.
+
+## Further reading
+
+* [See the other codebases we work with](https://publiccode.net/codebases/).
```
Comment (ElenaFdR) [inline _posts/2021-02-04-signalen-is-in-incubation.md:25]:
```suggestion
* [See the other codebases we work with](https://publiccode.net/codebases/)
```

**Review by ElenaFdR — APPROVED** [review]
Ready to go!

**Pull request merged**